src/test/java/org/opendevstack/provision/authentication/SimpleCachingGroupMembershipManagerTest.java

```diff
@@ -19,9 +19,9 @@
 import org.springframework.test.annotation.DirtiesContext;
 import org.springframework.test.context.ActiveProfiles;
 
+/** Testclass for the simple caching manager */
```

Comment:
This comment does not provide much help... it can be removed

Reply:
done